README.md

```diff
@@ -64,6 +64,8 @@ require "bump/tasks"
 
     rake bump:patch
     rake bump:current
+    rake bump:patch COMMIT=true TAG=false BUNDLE=false
```

Comment:
documenting COMMIT=true might be kinda misleading since that's the default ... maybe also use false ?